sdks/java/io/rabbitmq/src/test/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIOTest.java

```diff
@@ -282,28 +424,4 @@ public void testWriteExchange() throws Exception {
       }
     }
   }
-
-  private static List<byte[]> generateRecords(int maxNumRecords) {
```

Comment:
I moved these to a utilities class, and replaced all one-off UTF-8/byte[] conversions with calls to these utility functions

Reply:
+1, it looks good to me.